pnpm-lock.yaml

```diff
@@ -42181,6 +42160,39 @@ packages:
       yargs-parser: 21.1.1
     dev: true
 
+  /ts-jest/29.1.1_gvmt7aj7nai5bnvsxmiksfugce:
```

Comment:
There was another PR that added a new ts-jest recently. Are we going to end up with 3 copies?